parm/config/gfs/config.ufs

```diff
@@ -87,15 +87,22 @@ fi
 # (Standard) Model resolution dependent variables
 case "${fv3_res}" in
     "C48")
+        if [[ "${RUN}" = *gdas ]]; then
+          export layout_x=1
+          export layout_y=1
+          export nthreads_fv3=1
+          export nthreads_ufs=1
+          export WRITE_GROUP=1
+          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
+        elif [[ "${RUN}" = *gfs ]]; then
+          export layout_x=1
+          export layout_y=1
+          export nthreads_fv3=1
+          export nthreads_ufs=1
+          export WRITE_GROUP=1
+          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
+        fi
```

Comment:
Since these are identical, there's no need for an if-else:
```suggestion
        export layout_x=1
        export layout_y=1
        export nthreads_fv3=1
        export nthreads_ufs=1
        export WRITE_GROUP=1
        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
```